src/pca_gophish.yml

```
@@ -52,12 +52,18 @@
             name: "{{ gophish_group }}"
             gid: 421
             system: yes
-        - name: Change Gophish published port from 3380 to 80
+        - name: Change Gophish published port from 3380 to 443
           ansible.builtin.lineinfile:
             path: "{{ pca_gophish_composition_path }}/docker-compose.yml"
             regexp: "published: 3380$"
-            line: "        published: 80"
+            line: "        published: 443"
             state: present
+        - name: Configure gophish to use TLS
```

Capitalization nitpick:
```suggestion
        - name: Configure Gophish to use TLS
```